reanalysis/interpretation_runner.py

```diff
@@ -332,7 +332,7 @@ def main(
         # generate the jobs which run VEP & collect the results
         vep_jobs = add_vep_jobs(
             b=get_batch(),
-            input_siteonly_vcf_path=to_path(input_path),
+            vcf_path=to_path(input_path),
```

Comment:
https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/jobs/vep.py#L30 ?
```suggestion
            input_siteonly_vcf_path=to_path(input_path),
```

Reply:
Sorry 🤦